src/main/java/org/mastodon/mamut/project/MamutProject.java

```diff
@@ -204,6 +206,8 @@ public interface ProjectWriter extends Closeable
 		OutputStream getFeatureOutputStream( String featureKey ) throws IOException;
 
 		OutputStream getGuiOutputStream() throws IOException;
+
+		OutputStream getDatasetXmlFileOutputStream() throws IOException;
```

Comment:
Please rename to `getDatasetXmlOutputStream` to be more consistent with the other methods

Reply:
Done.